src/logic/contracts/safeContracts.js

```diff
@@ -10,19 +10,61 @@ import { calculateGasOf, calculateGasPrice } from '~/logic/wallets/ethTransactio
 import { ZERO_ADDRESS } from '~/logic/wallets/ethAddresses'
 
 export const SENTINEL_ADDRESS = '0x0000000000000000000000000000000000000001'
+export const MULTI_SEND_ADDRESS = '0xB522a9f781924eD250A11C54105E51840B138AdD'
+export const SAFE_MASTER_COPY_ADDRESS = '0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F'
+export const DEFAULT_FALLBACK_HANDLER_ADDRESS = '0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44'
+export const SAFE_PROXY_FACTORY_ADDRESS = '0x76E2cFc1F5Fa8F6a5b3fC4c8F4788F0116861F9B'
 
 let proxyFactoryMaster
 let safeMaster
 
 const createGnosisSafeContract = (web3: any) => {
-  const gnosisSafe = contract(GnosisSafeSol)
+  const gnosisSafeSol = { ...GnosisSafeSol }
```

Comment:
Why do we need to hardcode the addresses now and cannot take the ones from the npm package?

Reply:
the artifact doesn't have the addresses

![image](https://user-images.githubusercontent.com/3315606/74170035-06c2c380-4c0b-11ea-9759-adb4aa137e43.png)

Reply:
I see, this s a bug on the safe-contracts repo, please create an issue I will update it tomorrow